Source/NSFileManager.m

```diff
@@ -361,7 +361,7 @@ - (void) dealloc
   return _delegate;
 }
 
-- (void) setDelegate: (NSFileManager *)delegate {
```

Comment:
I think this error has already been corrected in master. Please merge and verify.